py-core/app/util/database_functions.py

```diff
@@ -0,0 +1,97 @@
+import sqlite3
+
+DATABASE_PATH = "./node_database.db"
+
+
+def setup_database():
+    database = sqlite3.connect(DATABASE_PATH)  # opens existing file or it makes new one if it does not exit
+    cursor = database.cursor()
+    command = """CREATE TABLE IF NOT EXISTS 'nodes' (
+        name VARCHAR PRIMARY KEY,
+        rpc_port INTEGER,
+        exposition_port INTEGER,
+        conseil_port INTEGER,
+        arronax_port INTEGER,
+        postgres_port INTEGER,
+        network VARCHAR,
+        history_mode VARCHAR,
+        status VARCHAR 
+        );"""
+    cursor.execute(command)
+    database.commit()
+    print("Database Setup")
```

Comment:
Please use a logging library. This, for example, should be something like this:

```python
logger.debug("Database Setup")
```

Reply:
This has since been removed, and a more in depth discussion should probably take place on logging/how it should be configured

Swap sent some tools over my way to look at in this regard

Reply:
Nevermind logging capability is added/will be expanded upon